### PR TITLE
Import Instruction

### LINKED ISSUE #56: Import Instruction
**As a** program user
**I want** to be able to use multi-file programs, importing code from other files  
**So that** my code can be split into multiple files.

**Request Description**
Implement an "import" instruction that can import code from other files on the system.  
We need to support both absolute paths and relative paths. We also need the capability to disable relative paths for the web-application.  

`import "/path/to/file.ez"`  
`import "relative/path/to/file.ez"`  

- You can use the existing simulation::Program struct to store data about the files already imported.  
- Running the same import line again should do NOTHING on successive calls.  
- The initial/main file will have a file ID of 0 and will be the entry point.  
- File IDs should increase by 1 with each new code file imported.  
- Importing a file which does not exist should result in a "File Not Found Error" of some sort.  
- Files should be able to be imported on windows using `/`.
- We will need to test this thoroughly on both Windows, Mac, and Linux.

### PR DESCRIPTION
**Pull request by TrevorBrunette**
Implements the import instruction for CLI application use
Closes #56

### CODE REVIEW

File: rezasm-source/rezasm-core/src/instructions/implementation/function_instructions.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+use crate::util::word_size::WordSize;
+
+lazy_static! {
+    pub static ref IMPORT: Instruction =
+        instruction!(import, |simulator: Simulator, input: InputTarget| {
+            let address = input.get(&simulator)?.int_value() as usize;
+            let file_path = simulator.get_memory().get_string(address)?;
+            let mut relative_location = simulator.get_program().main_file();
+            let relative_location_split = relative_location =
+                match relative_location.rsplit_once('/') {
```
Comment (WyattRoss) [inline rezasm-source/rezasm-core/src/instructions/implementation/function_instructions.rs:24]: No windows support? Do not merge yet

**Review by WyattRoss — CHANGES_REQUESTED** [review]
Support Windows.
Support WebAssembly.

File: rezasm-source/rezasm-core/src/instructions/implementation/function_instructions.rs (hunk 1 above)
Comment (TrevorBrunette) [reply rezasm-source/rezasm-core/src/instructions/implementation/function_instructions.rs:34]: Windows does work -- `/` as a file separator works fine. On tauri right now you have to do something like `import "C:/Users/.../code.ez"` to import code on Windows, but that is good behavior and should be documented. We should be good to merge.

**Review by WyattRoss — APPROVED** [review]
Ok(())

**Pull request merged**